src/model/errors/R2Error.ts

```diff
@@ -1,9 +1,15 @@
+interface R2ErrorExtraAction {
```

Comment:
Naming nitpick: the interface is used only internally in this module so the R2Error prefix is a bit redundant. "Extra" is also somewhat redundant IMO. Would "Action" be just as descriptive? Other option would be "ButtonProps" oslt?